examples/huggingface/pytorch/text-generation/quantization/run_generation_cpu_woq.py

```diff
@@ -286,7 +286,7 @@
             calib_len=args.calib_len,
             lr=args.lr,
             minmax_lr=args.minmax_lr,
-            enable_quanted_input=args.enable_quanted_input,
```

Comment:
enable_quanted_input = not args.disable_quanted_input

Reply:
improved.